example/android/app/src/androidTest/java/com/revenuecat/purchases_flutter/EmbeddingV1ActivityTest.java

```diff
@@ -0,0 +1,17 @@
+package com.revenuecat.purchases_flutter;
+
+import androidx.test.rule.ActivityTestRule;
+
+import com.revenuecat.purchases_flutter_example.EmbeddingV1Activity;
+
+import dev.flutter.plugins.e2e.FlutterRunner;
+import org.junit.Rule;
+import org.junit.runner.RunWith;
+
+@RunWith(FlutterRunner.class)
+public class EmbeddingV1ActivityTest {
+    @Rule
+    public ActivityTestRule<EmbeddingV1Activity> rule =
+            new ActivityTestRule<>(EmbeddingV1Activity.class);
+
+}
```

Comment:
This test makes sure the old APIs are still supported. I copied this from migration docs https://flutter.dev/docs/development/packages-and-plugins/plugin-api-migration#testing-your-plugin